src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -323,15 +323,19 @@ const General = ({
           rules={[
             {
               required: true,
-              message: 'Please select your Access Point Profile city.',
+              message: 'Please select your Access Point Profile.',
             },
           ]}
         >
           <Select
             className={styles.Field}
             onChange={handleProfileChange}
-            placeholder="Select access point profile..."
+            placeholder="Select Access Point profile..."
             onPopupScroll={onFetchMoreProfiles}
+            showSearch
+            filterOption={filterOption}
+            onSearch={onSearchProfile}
+            notFoundContent={loadingProfiles && <Spin data-testid="loadingProfiles" size="small" />}
```

Comment:
this is going to show infinite loading if there are no results? use the prop `loading`